assemblies/features/standard/src/main/feature/feature.xml

```diff
@@ -630,6 +630,7 @@ update = manager
     </feature>
 
     <feature name="instance" description="Provide Instance support" version="${project.version}">
+        <bundle dependency="true">mvn:org.apache.commons/commons-compress/${commons-compress.version}</bundle>
```

Comment:
I would prefer to avoid `commons-compress` dependency, and directly use JDK `Zip*` classes.

I can update the PR if you want.

Reply:
I'm happy for you to update it :+1:  Is there a tar option in JDK? There is `java.util.zip.GZIPOutputStream` and `java.util.zip.ZipOutputStream` but I don't know anything native for tarballs?

Reply:
There's no `tar` direct support in JDK. But a `tar` file is a uncompress kind of concat files with some headers. It's possible to create directly this file in Java, respecting the headers, offset, etc. It's basically what `commons-compress` is doing in the `tar` archiver.
As `instance:package` is an extra feature, I think it's fine to do just a zip archive (using `Zip*` JDK level classes).

Reply:
Thoughts ?

Reply:
@awrb any update about removing `commons-compress` and using only zip ? I can do that if you prefer. Thanks !

Reply:
I will modify the .zip today but not sure about .tar.gz. I'll look into the code of `commons-compress` how to do it. I'll ask you to take over if it takes too long.

Reply:
I think it's fine to only generate zip archive (not need to create tar.gz).

Reply:
I removed `commons-compress` dependency and just left the zip option. Sadly maintaining file permissions (e.g. 0755 in bin/karaf) does not seem to work. Seems to be a bug/limitation in JDK:
https://bugs.openjdk.org/browse/JDK-6194856
https://stackoverflow.com/questions/10699334/change-permission-of-a-zipentry

Reply:
Yes, it's normal in a zip file. Let's start with this first functionnality, we will improve on user requests.